src/functionalTest/groovy/com/autonomousapps/jvm/projects/IncludedBuildWithAnnotationProcessorProject.groovy

```diff
@@ -0,0 +1,126 @@
+package com.autonomousapps.jvm.projects
+
+import com.autonomousapps.AbstractProject
+import com.autonomousapps.kit.*
+import com.autonomousapps.model.Advice
+import com.autonomousapps.model.ProjectAdvice
+
+import static com.autonomousapps.AdviceHelper.*
+
+final class IncludedBuildWithAnnotationProcessorProject extends AbstractProject {
+
+  final GradleProject gradleProject
+
+  IncludedBuildWithAnnotationProcessorProject() {
+    this.gradleProject = build()
+  }
+
+  private GradleProject build() {
+    def builder = newGradleProjectBuilder()
+    builder.withRootProject { root ->
+      root.settingsScript.additions = """\
+        includeBuild 'processor-build'
+        include 'user-of-processor'
```

Comment:
good names, thanks

Reply:
Wasn't me ;) - this is taken (and adjusted) from #896